query-engine/connectors/mongodb-query-connector/src/root_queries/mod.rs

```diff
@@ -72,13 +66,9 @@ where
     histogram!(PRISMA_DATASOURCE_QUERIES_DURATION_HISTOGRAM_MS, elapsed);
     increment_counter!(PRISMA_DATASOURCE_QUERIES_TOTAL);
 
-    let params: Vec<i32> = Vec::new();
-
-    // todo: emit tracing event with the appropriate log level only query_log is enabled. And fix log suscription
-    if let Some(qs) = query_string_builder {
-        let qs = qs.build();
-        debug!(target: "mongodb_query_connector::query", item_type = "query", is_query = true, query = %qs, params = ?params, duration_ms=elapsed);
-    }
+    // TODO: emit the event only when the query logs are enabled.
```

Comment:
That TODO rewording removes a lot of the info that was initially put. Any reason why you removed a lot of it?

Reply:
I was meaning to ask you about it, but totally forgot. Do you know what the author meant by "fix log subscription"?

I propose to reword it to:
```
// TODO: emit the event only when "debug" level query logs are enabled.
```

Let me know if you have a better version in mind.